feat: add support for profile-set-once in Mixpanel integration

serkanhaslak opened the pull request:
Introduce a new feature to set user profile properties only if they are not already set. This includes adding a new option in the template and implementing the `sendSetOnceProfileRequest` function to handle the new profile-set-once request type.

serkanhaslak committed:
feat: add support for profile-set-once in Mixpanel integration

Introduce a new feature to set user profile properties only if they are not already set. This includes adding a new option in the template and implementing the `sendSetOnceProfileRequest` function to handle the new profile-set-once request type.

giovaniortolani commented on template.tpl:
```diff
@@ -477,6 +483,58 @@ function sendAppendProfileRequest() {
 
+function sendSetOnceProfileRequest() {
+    const userProperties = {};
+    data.userPropertiesToSetOnce.forEach(row => {
```
The `userPropertiesToSetOnce` field was not added to the UI. Please, add it.

<img width="740" alt="image" src="https://github.com/user-attachments/assets/6c0102bf-1652-4e2e-9e0a-77e8dc11b5fc" />

giovaniortolani reviewed (changes requested):
Please, fix:
- [this](https://github.com/stape-io/mixpanel-tag/pull/11/files/c5aa2b93b95bf118f4a0a6db21ce4cfa6d179cf4#r2076267630).
- Update the `template.js` file with the changes.
- Update the `metadata.yaml` file.

Let me know if you want me to do it instead.

serkanhaslak committed:
feat(profile): add support for 'set-once' user profile properties

Introduce new functionality to handle 'set-once' user profile properties in the Mixpanel integration. This includes adding a new UI configuration group for setting these properties and implementing the corresponding backend logic to send the data to Mixpanel. The changes enhance the flexibility and usability of the profile management feature.

serkanhaslak commented:
Hi Giovani, 

I've updated with your requests. Can you please check it?

giovaniortolani commented:
> Hi Giovani,
> 
> I've updated with your requests. Can you please check it?

Hi @serkanhaslak, thanks for the changes.

I had to fix the `metadata.yaml` because it wasn't in the required format/standard. Other than that, all good.

I'll push the changes in the #12. Took the opportunity to do some cleanup.
